error-prone-contrib/src/main/java/tech/picnic/errorprone/refasterrules/ImmutableListMultimapRules.java

```diff
@@ -276,4 +279,26 @@ ImmutableListMultimap<K, V2> after(
       return ImmutableListMultimap.copyOf(Multimaps.transformValues(multimap, transformation));
     }
   }
+
+  /**
+   * Prefer {@link ImmutableListMultimap.Builder#put(Object, Object)} over {@link
+   * ImmutableListMultimap.Builder#putAll(Object, Iterable)} when adding a single value.
+   */
+  static final class ImmutableListMultimapBuilderPutOverPutAllSingleValue<K, V> {
+    @BeforeTemplate
+    @SuppressWarnings("ImmutableListOf1")
+    ImmutableListMultimap.Builder<K, V> before(
+        ImmutableListMultimap.Builder<K, V> builder, K key, V value) {
+      return Refaster.anyOf(
+          builder.putAll(key, ImmutableList.of(value)),
```

Comment:
We can also add the varags overloaded variant:
```suggestion
          builder.putAll(key, value),
```